woocommerce/google-listings-and-ads
# Centralize frontend images to the dedicated directory

### Changes proposed in this Pull Request:

When processing tasks related to auditing images, image files scattered in different locations make the process cumbersome.

For future maintainability, this PR centralizes frontend end images to the dedicated directory so that it will be easier to find them.

### Detailed test instructions:

1. Run `npm run dev` or `npm run start` to see if the build is successful
2. View the code changes if see if the new paths are correct
3. Search the codebase to find if there is any image under `js/src` was missed to move to `js/src/images`

### Changelog entry

> Dev - Centralize frontend images to the dedicated directory.
